src/contexts/TransactionContext.tsx

```diff
@@ -69,12 +70,14 @@ export function TransactionContextProvider(props: TransactionContextProviderProp
     payloadHex: null
   });
 
+  useTransactionsStatus(transactionsState.transactions, dispatchTransaction);
```

Comment:
I have moved the hook to the directory `hooks/context` so at least at dir level, there is a relation between them.
Then regarding moving the `useReducer` to specific hook i'm not 100% convinced since now we have just one hook that updates the data model, but what happens if in the future we need to add more? which one should be the one defining the reducer?